### PR TITLE
CAL-310 Timezone difference causing availability day shift Bug fix

### PR DESCRIPTION
**Pull request by alishaz-polymath**
* Changes made to the `lib/slots.ts` file which seemed to have fixed the problem. Further testing is still required by another reviewer.

### CODE REVIEW

**Review by pumfleet — APPROVED** [review]
The code makes sense to me, but will wait for @emrysal's approval before merging because my knowledge of timezone knowledge is very limited 😁

**Review by emrysal — APPROVED** [review]

**Pull request merged**
